components/header.component.js

```diff
@@ -10,17 +8,6 @@ export default function({ transparent }) {
   return (
     <nav className="nav" style={styles.container}>
       <div className="nav-left">
-        <div className="nav-item is-brand" style={styles.text}>
-          <Link prefetch href="/">
```

Comment:
Have you checked the careers page? It might be pulling this icon into that view. The neosavvy icon at the top left is currently the only way of going back to the landing page from any of the careers page